include/kalmar_math.h

```diff
@@ -1538,91 +1540,91 @@ namespace Kalmar
         HCC_MATH_LIB_FN
         void sincos(double x, double *s, double *c)
         {
-            *s = __hc_sincos_double(x, c);
+            *s = __ocml_sincos_f64(x, c);
         }
 
         HCC_MATH_LIB_FN
-        float sinhf(float x) { return __hc_sinh(x); }
+        float sinhf(float x) { return __ocml_sinh_f32(x); }
 
         HCC_MATH_LIB_FN
-        __fp16 sinh(__fp16 x) { return __hc_sinh_half(x); }
+        __fp16 sinh(__fp16 x) { return __ocml_sinh_half(x); }
 
         HCC_MATH_LIB_FN
         float sinh(float x) { return precise_math::sinhf(x); }
 
         HCC_MATH_LIB_FN
-        double sinh(double x) { return __hc_sinh_double(x); }
+        double sinh(double x) { return __ocml_sinh_f64(x); }
 
         HCC_MATH_LIB_FN
-        float sqrtf(float x) { return __hc_sqrt(x); }
+        float sqrtf(float x) { return __ocml_sqrt_f32(x); }
 
         HCC_MATH_LIB_FN
-        __fp16 sqrt(__fp16 x) { return __hc_sqrt_half(x); }
+        __fp16 sqrt(__fp16 x) { return __ocml_sqrt_half(x); }
 
         HCC_MATH_LIB_FN
         float sqrt(float x) { return precise_math::sqrtf(x); }
 
         HCC_MATH_LIB_FN
-        double sqrt(double x) { return __hc_sqrt_double(x); }
+        double sqrt(double x) { return __ocml_sqrt_f64(x); }
 
         HCC_MATH_LIB_FN
-        float tgammaf(float x) { return __hc_tgamma(x); }
+        float tgammaf(float x) { return __ocml_tgamma_f32(x); }
 
         HCC_MATH_LIB_FN
-        __fp16 tgamma(__fp16 x) { return __hc_tgamma_half(x); }
+        __fp16 tgamma(__fp16 x) { return __ocml_tgamma_half(x); }
 
         HCC_MATH_LIB_FN
         float tgamma(float x) { return precise_math::tgammaf(x); }
```

Comment:
replace this with the __ocml_tgamma_f32
In fact, please replace all the ones with the precise_math namespace with an ocml equivalent